mod-source-record-storage-server/src/main/resources/liquibase/tenant/scripts/v-5.10.0/2025-02-07--15-00-create-composite-records-index.xml

```diff
@@ -0,0 +1,12 @@
+<databaseChangeLog
```

Comment:
please include this script in the changelog.xml file so that this index will be created

Reply:
@RuslanLavrov  Good catch! Thank you